ml/backend/ggml/ggml.go

```diff
@@ -530,6 +539,24 @@ func (c Context) Compute(tensors ...ml.Tensor) {
 	}
 }
 
+func (c Context) Reserve() error {
```

Comment:
A GoDoc comment here would be great! Since there are some preconditions to running this.